aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -42,19 +42,14 @@ struct MockKernel final : OperatorKernel {
 TEST(OperatorRegistrationTest, whenRegisteringSameSchemaWithAliasAnalysisAfterRegisteringWithoutAliasAnalysis_thenCanBeCalled) {
   {
     auto registrar1 = c10::RegisterOperators().op("_test::dummy(Tensor dummy) -> ()", c10::RegisterOperators::options().kernel<DummyKernel>(c10::DispatchKey::CPUTensorId));
-    auto registrar2 = c10::RegisterOperators().op("_test::dummy(Tensor dummy) -> ()", c10::RegisterOperators::options().kernel<DummyKernel>(c10::DispatchKey::XLATensorId).aliasAnalysis(at::AliasAnalysisKind::PURE_FUNCTION));
```

Comment:
I think this is not doing what the test name says anymore.

Reply:
I think I just need to rename the test name here

Reply:
split this into two tests and lined up the names